app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -340,6 +340,174 @@ def test_invalid_experiments_with_specific_languages(
+                context={"user": self.user},
+            ).data,
+            context={"user": self.user},
+        )
+        self.assertTrue(serializer_1.is_valid())
+
+    @parameterized.expand(
+        [
+            (
+                NimbusExperiment.Application.FOCUS_ANDROID,
+                NimbusExperiment.Version.FIREFOX_102,
+            ),
+            (
+                NimbusExperiment.Application.FENIX,
+                NimbusExperiment.Version.FIREFOX_102,
+            ),
+            (
+                NimbusExperiment.Application.IOS,
+                NimbusExperiment.Version.FIREFOX_101,
+            ),
+            (
+                NimbusExperiment.Application.FOCUS_IOS,
+                NimbusExperiment.Version.FIREFOX_101,
+            ),
+        ]
+    )
+    def test_valid_experiments_supporting_countries_versions_selecting_specific_country(
+        self, application, firefox_version
+    ):
+        # selected countries
+        country = CountryFactory.create()
+        experiment_1 = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+            application=application,
+            channel=NimbusExperiment.Channel.RELEASE,
+            firefox_min_version=firefox_version,
+            countries=[country.id],
+        )
+
+        serializer_1 = NimbusReviewSerializer(
+            experiment_1,
+            data=NimbusReviewSerializer(
+                experiment_1,
+                context={"user": self.user},
+            ).data,
+            context={"user": self.user},
+        )
+        self.assertTrue(serializer_1.is_valid())
+
+    @parameterized.expand(
+        [
+            (
+                NimbusExperiment.Application.FOCUS_ANDROID,
+                NimbusExperiment.Version.FIREFOX_101,
+            ),
+            (
+                NimbusExperiment.Application.FENIX,
+                NimbusExperiment.Version.FIREFOX_101,
+            ),
+            (
+                NimbusExperiment.Application.IOS,
+                NimbusExperiment.Version.FIREFOX_100,
+            ),
+            (
+                NimbusExperiment.Application.FOCUS_IOS,
+                NimbusExperiment.Version.FIREFOX_100,
+            ),
+        ]
+    )
+    def test_valid_experiments_with_all_countries(self, application, firefox_version):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+            application=application,
+            channel=NimbusExperiment.Channel.RELEASE,
+            firefox_min_version=firefox_version,
+            countries=[],
+        )
+
+        serializer = NimbusReviewSerializer(
+            experiment,
+            data=NimbusReviewSerializer(
+                experiment,
+                context={"user": self.user},
+            ).data,
+            context={"user": self.user},
+        )
+        self.assertTrue(serializer.is_valid())
```

Comment:
So isn't this a duplicate of `test_valid_experiments_supporting_countries_versions_default_as_all_countries` above now?

Reply:
Kind of no, I was thinking in this way- the above one includes valid versions for the mobile client, whereas this includes invalid versions but this shows that you can select all countries by default for this.

Reply:
Ah yes, then I would included something like `with_language_unsupported_version` in the name 🙏

Reply:
okey dokey, all set 🚀 , thank you 🙏